### PR TITLE
Chat worshop - corrections and improvements

### PR DESCRIPTION
**Pull request by Saul-BT**
**Summary:**
- Custom topic to join swarm (with 'crypto') as [hyperswarm readme says](https://github.com/hyperswarm/hyperswarm/blob/master/README.md). (related to problem 8).
- Some typo, format and output improvements.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,10 @@
   "keywords": [],
   "dependencies": {
     "chalk": "^3.0.0",
+    "crypto": "^1.0.1",
```
Comment (hackergrrl) [inline package.json]: This is already a built-in node module.
```suggestion
```

File: solutions/07b/single-chat.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ feed.append({
   nickname: 'cat-lover',
   text: 'hello world',
   timestamp: '2018-11-05T14:26:000Z' // new Date().toISOString()
-}, function (err, seq) {
+}, function(err, seq) {
```
Comment (hackergrrl) [inline solutions/07b/single-chat.js]: I noticed that you changed the formatting throughout of
```js
function (args)
```
to
```js
function(args)
```
Can you please maintain this convention?

**Review by hackergrrl — CHANGES_REQUESTED** [review]
Thank you for these revisions :rocket:

File: solutions/07b/single-chat.js (hunk 2 above)
Comment (okdistribute) [reply solutions/07b/single-chat.js]: If you're wondering where it comes from, it is standard style -- the `standard` cli tool has a feature that automatically format it for you in this style

Comment (Saul-BT) [reply solutions/07b/single-chat.js]: Sorry, I hadn't noticed. I use tide-format to format my code in emacs.
Change is really necessary?

Comment (Saul-BT) [reply solutions/07b/single-chat.js]: Ok @okdistribute, I'll reformat it

Comment (Saul-BT) [reply solutions/07b/single-chat.js]: Reformatted d021a4f4b8aa7e6c61c06206af92b30556d519bc

**Pull request merged**